pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
```

Comment:
I'm not sure if flake8 natively supports `pyproject.toml`, but if you find any documentation, please let me know and I'll update it. In the meanwhile, I'll take care of the isort configuration update.

Reply:
@vanessa You're right - my mistake. It's just `isort` that you can put in here. I thought I'd put both in, but it's been a while since I'd done it so just mis-remembering.